pkg/resources/resource_monitor.go

```diff
@@ -283,13 +277,54 @@ func extractTriggerInts(s sql.NullString) ([]int, error) {
 	return out, nil
 }
 
+// inSlice returns true if n is in string slice h otherwise false.
+func inSlice(n string, h []string) bool {
```

Comment:
there is a slices.Contains() function you should use instead of rolling your own function https://pkg.go.dev/golang.org/x/exp/slices#Contains